ZDF_2_CSV.py

```diff
@@ -0,0 +1,52 @@
+"""
+This example shows how to convert a Zivid point cloud from a .ZDF file format
+to a .CSV file format.
+"""
+
+import zivid
+import numpy as np
+from math import ceil
+
+
+def _main():
+
+    app = zivid.Application()
+
+    # The Zivid3D.zdf file has to be in the same folder as this sample script.
+    FilenameZDF = "Zivid3D.zdf"
+    FilenameCSV = "Zivid3D.csv"
+
+    print("Reading ", FilenameZDF, " point cloud")
+    frame = zivid.Frame(FilenameZDF)
+
+    pc = frame.get_point_cloud()
+
+    # Disorganizing the point cloud
+    pc = np.dstack(
+        [pc["x"], pc["y"], pc["z"], pc["r"], pc["g"], pc["b"], pc["contrast"]]
+    )
+    pts = pc.reshape(-1, 7)
+    # Just the points without color and contrast
+    # pc = np.dstack([pc['x'],pc['y'],pc['z']])
```

Comment:
Is it common to have commented out code in samples? Makes the samples longer and more noisy